Makefile

```diff
@@ -35,26 +44,48 @@ clean: \
 check-syntax-errors: .
 	@$(call help,$@:)
 	@python -m compileall -q $?
+	@$(call pass,$@:)
+
+## Run linters to check for syntax and style errors.
+check-style: \
+	check-style-all \
+	check-style-doc-commons \
+	check-style-doc-types
+	@$(call pass,$@:)
 
 ## Run linters to check for syntax and style errors.
-check-style: $(shell git ls-files "*.py")
+check-style-all:
 	@$(call help,$@:)
-	@flake8 $?
+	@flake8 `git ls-files | grep "\.py$$"`
 
-## Run code formatters to correct style errors.
-format-style: $(shell git ls-files "*.py")
+## Run linters to check for syntax and style errors.
+check-style-doc-%:
 	@$(call help,$@:)
-	@autopep8 $?
+	@flake8 --select=D101,D102,D103,DAR openfisca_core/$*
+	@pylint --enable=classes,exceptions,imports,miscellaneous,refactoring --disable=W0201,W0231 openfisca_core/$*
```

Comment:
Isn't the configuration for `flake8` and `pylint` already managed in a config file? If not, this is acceptable for me as RFC #1040 says source of truth should be the task manager.

Reply:
Well, this are transitional for me, indeed I can move them to a specific task just for the maintainers, following consensus of #1040. At least, it is not —yet— my intention to impose them, even if those rules are a subset of what's already on country-template.

Reply:
Added a comment on why these are here, TL; DR: doing otherwise would force the whole codebase to be linted, diminishing the chances of these improvements to ever be integrated.